### DISCUSSION TITLE
ocvalidate.exe fails on unmodified config.plist

### DISCUSSION THREAD

**Discussion by ws1088** [tylernguyen/x1c6-hackintosh#167] [Q&A] [answered]
Hi, I tried to use ocvalidate.exe on the unmodified `config.plist` in the repository and it gives the following issues:
```
OCS: Missing key PasswordHash, context <Security>!
OCS: Missing key PasswordSalt, context <Security>!
OCS: Missing key MLB, context <Generic>!
OCS: Missing key ROM, context <Generic>!
OCS: Missing key SystemSerialNumber, context <Generic>!
OCS: Missing key SystemUUID, context <Generic>!
Serialisation returns 6 errors!

Completed validating config.plist in 27 ms. Found 6 issues requiring attention.
```
Is this expected?

**Comment by tylernguyen** [comment, marked as answer]
@ws1088 

Those errors are to be expected. What `ocvalidate` is saying here is that `config.plist` is missing PlatformInfo for iServices. These info include fake machine serial and should be generated on per machine/user basis.

See https://dortania.github.io/OpenCore-Post-Install/universal/iservices.html
